qiskit/providers/ibmq/runtime/runtime_program.py

```diff
@@ -177,13 +180,13 @@ def description(self) -> str:
         return self._description
 
     @property
-    def parameters(self) -> List['ProgramParameter']:
+    def parameters(self) -> 'ParameterNamespace':
```

Comment:
Is this supposed to be a method or property? You have `@property` here but the example has a method. If it's a property it should be `parameter` (singular) to be consistent with the other auto-completable code in this package.

Reply:
Fixed!